app/views/users/show.html.haml

```diff
@@ -0,0 +1,44 @@
+%header.entry-header
+  - if @user.admin?
+    %h1.entry-title.is-admin=  @user.email
+  - else
+    %h1.entry-title=  @user.email
+
+
+.entry-content
+  %p
+    %b #{t('.email')}:
+    = @user.email
+
+  - if @user.last_sign_in_at.blank? && @user.current_sign_in_at.blank?
+    %p  #{@user.email} #{t('.user_has_never_signed_in')}
+
+  - else
+    %p
+      %b #{t('.last_login')}:
+      #{ most_recent_login_time @user}
+      (#{time_ago_in_words(most_recent_login_time @user)} ago)
+    %p
+      %b #{t('.last_login_ip')}:
+      #{ most_recent_login_ip @user}
+
+
+
+    %p
+      %b #{t('.logged_in_count')}:
+      = @user.sign_in_count
+
+  - if @user.admin?
+    %p.is-admin #{t('.is_an_admin')}
```

Comment:
I'd like to leave this in because it gives us the ability to apply styling.  We can  easily define different styles for the email at the top and this one.  That would just need a change to the .css file.  (And of course it could always be the case that no styling is applied to this at all.)

(The `index` list should actually use a CSS class instead of using the span.  But it's easy enough to change later.  That's a simple story that would be good for a novice, I think.)